bloom-housing/bloom
# feat: adding community type disclaimer fields

This PR addresses https://github.com/bloom-housing/bloom/issues/4478


## Description
This creates new community type disclaimer fields in the db and api

## How Can This Be Tested/Reviewed?
from the api you should be able to create a listing with the new fields set and they will save, same for updating an existing listing

## Author Checklist:


## Review Process:

- Read and understand the issue
- Ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Either (1) explicitly ask a clarifying question, (2) request changes, or (3) approve the PR, even if there are very small remaining changes, if you don't need to re-review after the updates
